consumer/processing.go

```diff
@@ -211,35 +212,41 @@ func checkReportStructure(r Report) error {
 }
 
 // parseMessage tries to parse incoming message and read all required attributes from it
-func parseMessage(messageValue []byte) (incomingMessage, error) {
+func parseMessage(messageValue []byte) (incomingMessage, []types.ReportItem, error) {
```

Comment:
Why not including "rules" as a field of the `incomingMessage` type instead changing the number of returned valus, and in fact the rules array is part of the incoming message